baybe/recommenders/pure/bayesian/base.py

```diff
@@ -79,7 +82,15 @@ def setup_acquisition_function(
             searchspace: The search space in which the experiments are to be conducted.
             train_x: The features of the conducted experiments.
             train_y: The corresponding response values.
+
+        Raises:
+            NotImplementedError: If the setup is attempted from empty training data
         """
+        if train_x is None or train_y is None:
+            raise NotImplementedError(
+                "Bayesian recommenders do not support empty training " "data yet."
```

Comment:
The " signs look weird in this string, is this intended?

Reply:
nope, Gut Spot, fixed